BUILDING.md

```diff
@@ -127,7 +127,7 @@ Install [Homebrew](https://brew.sh/) if not already installed.
 #### Install Required Packages
 
 ```bash
-brew install cmake pkg-config go@1.22 qt@5 jq nvm yarn protobuf
+brew install cmake pkg-config go@1.23 qt@5 jq nvm yarn protobuf
```

Comment:
```suggestion
brew install cmake pkg-config go@1.23 qt@6 jq nvm yarn protobuf
```

Reply:
I think I will not touch QT instructions in this PR because there are many instances in this document below which also have instructions to set PATHs and everything, This seems like a job for another PR.